typed-racket-test/succeed/apply-dots.rkt

```diff
@@ -12,12 +12,6 @@
           (apply (lambda: ([x : Number] . [y : Number *]) x)
                  1 w))
 
-(plambda: (a ...) ([z : String] . [w : Number *])
```

Comment:
This no longer works due to the incompleteness mentioned above, but we're not worrying about that right now. Maybe it should go in `xfail` or in a comment here.